test/CSharpPrecompiledTestProjects/Dependency55/Dependency55.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.IdentityModel.Protocols" Version="5.5.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Required?

Reply:
Yes, this is a change that has been in `dev` for a bit. Forgot to port it over to `in-proc`.